<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Schedule_Sacrifice_Form</class>
 <widget class="QMainWindow" name="Schedule_Sacrifice_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schedule Sacrifice</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>-9</y>
      <width>591</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(G:/Final1/Image_ScheduleSacrifice.jpg) ;
background-repeat: no-repeat;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="Name">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="Day">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="Price">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="Price_Label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>Price:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Day_Label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>Day Of Sacrifice:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Days_Options">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Kristen ITC&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>Select</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Matura MT Script Capitals&quot;;
background-color:black;
color:white;</string>
    </property>
    <property name="text">
     <string>Schedule Sacrifice</string>
    </property>
   </widget>
   <widget class="QLabel" name="Name_Label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>Name: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Submit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0);
font: 20pt &quot;Bauhaus 93&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Schedule Sacrifice</string>
    </property>
   </widget>
   <widget class="QLabel" name="Enter_Label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>Enter Day of Sacrifice: </string>
    </property>
   </widget>
   <widget class="QComboBox" name="List">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Kristen ITC&quot;;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>583</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
